### DISCUSSION TITLE
Vulnerability PR ignores package rules

### DISCUSSION THREAD

**Discussion by daniluk4000** [renovatebot/renovate#30649] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

https://github.com/yandex-maps-unofficial/vue-yandex-maps/pull/354

https://github.com/yandex-maps-unofficial/vue-yandex-maps/blob/master/.github/renovate.json5#L15

We have a rule that should not update Nuxt 2 package, but security update still does that.

In my point of view, this could be a bug, because I can't find anything in config that we are doing wrong.

Thanks!

**Comment by rarkins** [comment]
Vulnerability updates use allowedVersions themselves so this is behaving as designed. If there are no v2 updates anticipated then you could disable that dependency

**Reply by daniluk4000** [reply]
Sorry, how? Can't find anything in docs for that

**Reply by rarkins** [reply]
Set enabled=false
